src/widgets/dialogs/SettingsDialog.cpp

```diff
@@ -193,9 +193,7 @@ void SettingsDialog::filterElements(const QString &text)
 
 void SettingsDialog::addTabs()
 {
-    this->ui_.tabContainer->setMargin(0);
```

Comment:
This was removed because it's useless as `setContentsMargins` is called below and overrides any margins this would've changed